library_generation/test/integration_tests.py

```diff
@@ -247,6 +294,8 @@ def __run_entry_point_in_docker_container(
                 f"{repo_location}:/workspace/repo",
                 "-v",
                 f"{config_location}:/workspace/config",
+                "-v",
+                f"{config_dir}/{WELL_KNOWN_GENERATOR_JAR_FILENAME}:/home/.library_generation/{WELL_KNOWN_GENERATOR_JAR_FILENAME}",
```

Comment:
I guess this would override the SNAPSHOT jar built in the previous step `docker build`?

Reply:
Yes, exactly. This is just to ensure that the outcome of the test doesn't depend on changes to the generator code.